.github/workflows/build_linux_wheel/action.yml

```diff
@@ -20,23 +21,37 @@ runs:
       shell: bash
       run: |
         echo "ARM BUILD: ${{ inputs.arm-build }}"
-    - name: Build manylinux image
+    - name: Build x86_64 Manylinux wheel
       if: ${{ inputs.arm-build == 'false' }}
-      shell: bash
-      working-directory: python
-      run: |
-        docker build -t pylance_manylinux -f tools/Dockerfile.manylinux2014  tools/
-    - name: Build manylinux image (arm)
-      if: ${{ inputs.arm-build == 'true' }}
-      shell: bash
-      working-directory: python
-      run: |
-        docker build -t pylance_manylinux -f tools/Dockerfile.manylinux2014_aarch64 tools/
-    - name: Build wheel
       uses: PyO3/maturin-action@v1
       with:
         command: build
+        working-directory: python
+        target: x86_64-unknown-linux-gnu
+        manylinux: "2_17"
         args: ${{ inputs.args }}
+        before-script-linux: |
+          yum install -y openssl-devel \
+            && curl -L https://github.com/protocolbuffers/protobuf/releases/download/v21.12/protoc-21.12-linux-$(uname -m).zip > /tmp/protoc.zip \
+            && unzip /tmp/protoc.zip -d /usr/local \
+            && rm /tmp/protoc.zip
+    - name: Build Arm Manylinux Wheel
+      if: ${{ inputs.arm-build == 'true' }}
+      uses: PyO3/maturin-action@v1
+      with:
+        command: build
         working-directory: python
-        container: pylance_manylinux
-        docker-options: "-e SOME_FLAG_TO_INVALIDATE_CACHE=1"
+        target: aarch64-unknown-linux-gnu
```

Comment:
Do we still need arm runner?

Reply:
Only for when we are running the tests. If we are just building we don't.